### PR TITLE
Redirection after language choice

### PR DESCRIPTION
**Pull request by vdusart**
Redirection after language choice.

## Description

After choosing the language, we had to come back to the page we were reading.
With this feature, the page from which we come is saved in a parameter `from` in the url and the choice of the language redirects us to this page but in the new language.

See bellow a flow of url:
```
https://ethereum.org/en/developers/docs/programming-languages/
https://ethereum.org/en/languages/?from=/developers/docs/programming-languages/
https://ethereum.org/fr/developers/docs/programming-languages/
```

## Related Issue

None

### CODE REVIEW

File: src/components/Nav/index.tsx [hunk 1]
```diff
@@ -97,25 +96,6 @@ const HomeLogo = styled(HomeIcon)`
...
-    svg {
-      fill: ${(props) => props.theme.colors.primary};
-    }
-  }
-`
-
-const NavIcon = styled(Icon)`
-  fill: ${(props) => props.theme.colors.text};
-`
-
```
Comment (pettinarip) [inline src/components/Nav/index.tsx:118]: Wondering why we have removed these, were they unused components?

Comment (pettinarip) [reply src/components/Nav/index.tsx:118]: If so, thanks :)

**Review by pettinarip — APPROVED** [review]
Cool feature @vdusart thanks for doing that

File: src/components/Nav/index.tsx (hunk 1 above)
Comment (vdusart) [reply src/components/Nav/index.tsx:118]: Yes they were all 3 unused :smile:

**Pull request merged**